cray_product_catalog/catalog_update.py

```diff
@@ -172,10 +187,16 @@ def update_config_map(data: dict, name, namespace):
             LOGGER.exception("Error calling read_namespaced_config_map")
 
             # ConfigMap doesn't exist yet
-            if err.status == ERR_NOT_FOUND:
+            if err.status != ERR_NOT_FOUND:
+                raise   # unrecoverable
+            if name == CONFIG_MAP:
+                # If main ConfigMap is not found wait until it is available
                 LOGGER.warning("ConfigMap %s/%s doesn't exist, attempting again", namespace, name)
-                continue
-            raise  # unrecoverable
+            else:
+                # If product ConfigMap is not available then create
+                LOGGER.warning("Product ConfigMap %s/%s doesn't exist, attempting to create", namespace, name)
```

Comment:
I think this will be the normal behavior when adding information for a new version of an existing product or for a new product. As such, a warning-level message will be misleading for the end user. This should probably be an info-level message.